src/three/GLTFCesiumRTCExtension.js

```diff
@@ -9,10 +8,14 @@ export default class GLTFCesiumRTCExtension {
 
 	afterRoot( res ) {
 
-		const { center } = res.parser.json.extensions.CESIUM_RTC;
-		res.scene.position.x += center[ 0 ];
-		res.scene.position.y += center[ 1 ];
-		res.scene.position.z += center[ 2 ];
+		if ( res.parser.json.extensions ) {
+
+			const { center } = res.parser.json.extensions.CESIUM_RTC;
+			res.scene.position.x += center[ 0 ];
+			res.scene.position.y += center[ 1 ];
+			res.scene.position.z += center[ 2 ];
+
+		}
```

Comment:
Can you explain the need for wrapping this in a condition? I'm less familiar with building glTF extensions so I'm just curious.

Reply:
Sure, when we register the CESIUM_RTC extension, GLTFLoader will attempt to run it against all GLTF even if they do not support CESIUM_RTC. 
So the above check ensures the properties required to run the extension are available, otherwise, we simply return as nothing needs to be done. Upon a second look, I should probably add a null check for the CESIUM_RTC property as well.

Reply:
Thanks for the explanation!

> I should probably add a null check for the CESIUM_RTC property as well.

This sounds good!

Reply:
@gkjohnson I have pushed the extra changes just to ensure the center property is non-null before accessing it this should leave us in a pretty safe position moving forward.

Reply:
Also, as I would love to pull and use the changes I've made, any ideas on when the next release will be? (0.3.15) Thanks!

Reply:
Thanks for the changes! I will merge and publish over the next couple days. I am in the middle of an international move so I'm moving a little slow online at the moment. Thanks for the patience!